packages/xs-extra/xapi-netdev.master/opam

```diff
@@ -12,8 +12,6 @@ install: [
   [make "install" "BINDIR=%{bin}%"]
 ]
 remove: [
-  ["oasis" "setup"]
-  [make "uninstall" "BINDIR=%{bin}%"]
```

Comment:
Oh, so running the `make uninstall` provided by OASIS isn't even necessary, is it sufficient to only run `ocamlfind remove`? https://github.com/xapi-project/xs-opam/pull/34/files and https://github.com/xapi-project/netdev/pull/8 fixed the uninstallation of `netdev` by adding the `oasis setup` line.